lib/internal/crypto/util.js

```diff
@@ -168,15 +171,6 @@ const kNamedCurveAliases = {
 
 const kAesKeyLengths = [128, 192, 256];
 
-// These are the only hash algorithms we currently support via
-// the Web Crypto API.
-const kHashTypes = [
-  'SHA-1',
-  'SHA-256',
-  'SHA-384',
-  'SHA-512',
-];
```

Comment:
This removal looks unrelated, IMO it would be clearer to move it to a separate PR. Happy to do that for you if you want.

Reply:
I find that to be a waste of reviewer's time and CI resources.

Reply:
Yet that is important so git blame stays comprehensible, and in case we need to revert

Reply:
If you want to drive that PR, please do so.